src/tests/ftest/util/nvme_utils.py

```diff
@@ -92,6 +95,18 @@ def setUp(self):
         self.engines = self.server_managers[0].manager.job.yaml.engine_params
         self.dmg_command = self.get_dmg_command()
 
+    def create_container(self):
+        """Create the container """
+        # Get container params
+        self.nvme_local_cont = TestContainer(self.pool, daos_command=DaosCommand(self.bin))
+        self.nvme_local_cont.get_params(self)
+
+        # update container oclass
+        if self.ior_local_cmd.dfs_oclass:
+            self.nvme_local_cont.oclass.update(self.ior_local_cmd.dfs_oclass.value)
+
+        self.nvme_local_cont.create()
```

Comment:
I'd recommend using this existing function:
https://github.com/daos-stack/daos/blob/2b0d4a638977a4f8c7a0566ea9a8160ed0bcd573/src/tests/ftest/util/apricot/apricot/test.py#L1669

```suggestion
        # Get container params
        self.nvme_local_cont = self.get_container(self.pool, create=False)

        # update container oclass
        if self.ior_local_cmd.dfs_oclass:
            self.nvme_local_cont.oclass.update(self.ior_local_cmd.dfs_oclass.value)

        self.nvme_local_cont.create()
```